tools/upgrade.sh

```diff
@@ -25,11 +25,14 @@ gateway_old=./gateway
 # node_modules
 cp -r $gateway_old /tmp/gateway
 if [ -d "/tmp/gateway/node_modules" ]; then
-	rm -r /tmp/gateway/node_modules
+  rm -r /tmp/gateway/node_modules
 fi
 
 die() {
   sudo systemctl start mozilla-iot-gateway.service
+  rm gateway-*.tar.gz
```

Comment:
These rm's should probably use rm -f so that they won't fail if the files don't exist for some reason